src/test/java/org/apache/platypus/server/YelpBaseTest.java

```diff
@@ -0,0 +1,72 @@
+package org.apache.platypus.server;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.util.JsonFormat;
+import org.apache.platypus.server.grpc.*;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.logging.Logger;
+
+public class YelpBaseTest {
+    protected static final Logger logger = Logger.getLogger(YelpReviewsTest.class.getName());
```

Comment:
rename?

Reply:
can me make the name  more representative of what this class does? For example `TestIndexManager`?